src/popup/router/pages/TipsClaim.vue

```diff
@@ -1,5 +1,9 @@
 <template>
   <div class="tips-claim">
+    <AccountInfo
+      v-bind="$attrs"
```

Comment:
In my opinion this case of use `v-bind="$attrs"` makes the code less readable and can lead to further problems. The `AccountInfo` components requires only 3 props and here the binding will pass all unnecessary stuff to it.